config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -961,7 +961,7 @@ periodics:
       - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
       env:
       - name: FEATURE_GATES
-        value: '{"AllAlpha":true,"AllBeta":true,"EventedPLEG": false}'
+        value: '{"AllAlpha":true,"AllBeta":true,"EventedPLEG": false,"ListFromCacheSnapshot": false}'
```

Comment:
we don't want different cluster behaviors depending on which features are enabled, the goal of this jobs is to ensure our cluster behavior is consistent

/hold

Reply:
also, this is going to be enabled in more jobs, I don't like the idea of start modifying all the jobs, can we think in a better solution before trying this?

Reply:
I think just disabling an alpha feature is better than changing conformance test during code freeze.

Reply:
then please add a comment explaining this and a TODO , we should fix this in 1.34

 Please also modify these jobs https://github.com/kubernetes/test-infra/pull/34584 , I've realized we were not running these tests on release informing and we missed this

/hold cancel

Reply:
@serathius disable the test, not the feature

```
Servers with support for API chunking should support continue listing from the last key if the original version has been compacted away, though the list is inconsistent 
```

Reply:
Thanks.